classwork_1.js

```diff
@@ -0,0 +1,32 @@
+// 1
+const cutString = (str, n) => {
```

Comment:
nitpick: don't shortcut names of arguments. `(string, charsQuantity)` looks more clear